app/assets/stylesheets/application.scss

```
@@ -0,0 +1,33 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+/* universal */
+body {
+  padding-top: 60px;
+  }
```

```suggestion
body {
  padding-top: 60px;
}
```
những chỗ dưới tương tự